include/aspect/material_model/rheology/drucker_prager.h

```diff
@@ -136,6 +141,22 @@ namespace aspect
           compute_derivative (const double angle_internal_friction,
                               const double effective_strain_rate) const;
 
+          /**
+           * Compute the LHS and RHS dilation terms for the Stokes system.
+           * LHS: $\bar\alpha\alpha / \eta^{ve}$;
+           * RHS: $\bar(2\eta^{ve}\varepsilon^{eff} - k) / \eta^{ve}$.
+           * Here $\alpha$ and $\bar\alpha$ correspond to the friction angle
+           * and the dilation angle, respectively, $k$ is cohesion,
+           * $\eta^{ve}$ is the non-yielding viscosity, and $\varepsilon^{eff}$
+           * is the effective viscosity.
+           */
+          std::pair<double,double>
+          compute_dilation_terms_for_stokes_system(const double angle_friction,
+                                                   const double angle_dilation,
+                                                   const double cohesion,
+                                                   const double non_yielding_viscosity,
+                                                   const double effective_strain_rate) const;
```

Comment:
could you use the following interface instead, this is more in line with that I did in #6384.
```suggestion
          compute_dilation_terms_for_stokes_system(const double angle_friction,
                                                   const double angle_dilation,
                                                   const double cohesion,
                                                   const double non_yielding_viscosity,
                                                   const double effective_strain_rate) const;
```